inc/editor/patterns/report.php

```diff
@@ -51,7 +51,7 @@
 <!-- /wp:heading -->
 
 <!-- wp:wmf-reports/table-of-contents {"waypoints":"[{\u0022tocLabel\u0022:\u0022Letter from the CEO\u0022,\u0022tocSlug\u0022:\u0022toc-letter-from-the-ceo\u0022},{\u0022tocLabel\u0022:\u0022By the Numbers\u0022,\u0022tocSlug\u0022:\u0022toc-by-the-numbers\u0022},{\u0022tocLabel\u0022:\u0022Stories\u0022,\u0022tocSlug\u0022:\u0022toc-stories\u0022},{\u0022tocLabel\u0022:\u0022Financial Statments\u0022,\u0022tocSlug\u0022:\u0022toc-financial-statments\u0022},{\u0022tocLabel\u0022:\u0022Leadership\u0022,\u0022tocSlug\u0022:\u0022toc-leadership\u0022},{\u0022tocLabel\u0022:\u0022Endowment and Legacy\u0022,\u0022tocSlug\u0022:\u0022toc-endowment-and-legacy\u0022},{\u0022tocLabel\u0022:\u0022Donors\u0022,\u0022tocSlug\u0022:\u0022toc-donors\u0022},{\u0022tocLabel\u0022:\u0022Previous Reports\u0022,\u0022tocSlug\u0022:\u0022toc-previous-reports\u0022}]"} -->
-<nav class="wp-block-wmf-reports-table-of-contents"><ul><li><a href="#toc-letter-from-the-ceo">Letter from the CEO</a></li><li><a href="#toc-by-the-numbers">By the Numbers</a></li><li><a href="#toc-stories">Stories</a></li><li><a href="#toc-financial-statments">Financial Statments</a></li><li><a href="#toc-leadership">Leadership</a></li><li><a href="#toc-endowment-and-legacy">Endowment and Legacy</a></li><li><a href="#toc-donors">Donors</a></li><li><a href="#toc-previous-reports">Previous Reports</a></li></ul></nav>
+<ul class="wp-block-wmf-reports-table-of-contents"><li><a href="#toc-letter-from-the-ceo">Letter from the CEO</a></li><li><a href="#toc-by-the-numbers">By the Numbers</a></li><li><a href="#toc-stories">Stories</a></li><li><a href="#toc-financial-statments">Financial Statments</a></li><li><a href="#toc-leadership">Leadership</a></li><li><a href="#toc-endowment-and-legacy">Endowment and Legacy</a></li><li><a href="#toc-donors">Donors</a></li><li><a href="#toc-previous-reports">Previous Reports</a></li></ul>
```

Comment:
I don't see any style or block changes associated with this - is losing the nav element intentional?

Reply:
Thanks for asking — yes, it is, because the block that's in use in production currently doesn't have the `nav` in its save method. I think we should evaluate adding that back in, but currently this change adapts the pattern to match the deployed version of the block.